score/cli.py

```diff
@@ -2,11 +2,11 @@
 
 import click
 import duckdb
-
 from .conda.get_conda_package_names import get_conda_package_names
 from .conda.scrape_conda import scrape_conda
 from .data_retrieval.json_scraper import scrape_json
 from .data_retrieval.web_scraper import scrape_web
+from .data_retrieval.github_scraper import scrape_github_data
```

Comment:
can you move this to a `.github` module?

```suggestion
from .github.github_scraper import scrape_github_data
```